vnext/Microsoft.ReactNative/ReactNativeAppBuilder.h

```diff
@@ -14,12 +17,14 @@ struct ReactNativeAppBuilder : ReactNativeAppBuilderT<ReactNativeAppBuilder> {
       winrt::Microsoft::UI::Composition::Compositor const &compositor);
   winrt::Microsoft::ReactNative::ReactNativeAppBuilder SetAppWindow(
       winrt::Microsoft::UI::Windowing::AppWindow const &appWindow);
-  winrt::Microsoft::ReactNative::ReactNativeHost Start();
+  winrt::Microsoft::ReactNative::ReactNativeAppBuilder SetReactViewOptions(
+      winrt::Microsoft::ReactNative::ReactViewOptions const &reactViewOptions);
+  winrt::Microsoft::ReactNative::ReactNativeWin32App Build();
 
  private:
-  winrt::Microsoft::UI::Windowing::AppWindow m_appWindow{nullptr};
-  winrt::Microsoft::UI::Composition::Compositor m_compositor{};
   winrt::Microsoft::ReactNative::ReactNativeHost m_host{};
+  winrt::Microsoft::ReactNative::ReactViewOptions m_reactViewOptions{};
```

Comment:
Why is this here and not in ReactNativeWin32App?

Reply:
For V1, Let this be here. I need to still figure out some of these dependencies on when I can set the reactViewOptions.